ql/src/java/org/apache/hadoop/hive/ql/ddl/table/AbstractAlterTableDesc.java

```diff
@@ -62,8 +62,8 @@ public AlterTableType getType() {
   }
 
   @Explain(displayName = "table name", explainLevels = { Level.USER, Level.DEFAULT, Level.EXTENDED })
-  public String getTableName() {
-    return tableName;
+  public String getDbTableName() {
```

Comment:
in this file we have "FullTableName" and "DbTableName" - I think it would help to define what is what in the TableName class or something; and the (try) to use the same concepts consequently  (of course this can be done later as well)

Reply:
Currently the 2 methods are returning the same value, however one adds an explain, while the other is inherited and is used in e.g. `Driver:acquireLocks()`. I'd say these are 2 separate use cases (e.g. I wouldn't want `acquireLocks` to generate that much output with the explain), so I'd rather just rename one of them to make it more self-explanatory.